test/testcafe/spec-en-leaks/EnglishLeaks_spec.js

```diff
@@ -53,7 +53,6 @@ const ignoredMocks = [
   'authenticator-enroll-email.json',
   'authenticator-enroll-data-phone.json',
   'authenticator-enroll-data-phone-voice.json',
-  'terminal-enduser-email-consent-denied.json',
```

Comment:
Mind remove the json from here https://github.com/okta/okta-signin-widget/blob/5.6/.eslintrc.js#L122 and see if the lint passes ?